src/cbc_sdk/platform/observations.py

```diff
@@ -192,6 +227,59 @@ def get_network_threat_metadata(self):
         except AttributeError:
             raise ApiError("No available network threat metadata.")
 
+    @staticmethod
+    def search_suggestions(cb, query, count=None):
```

Comment:
Thought it is tautology for the name to be `observations_search_suggestions`, but if you think it is better to use `observations_search_suggestions`